aten/src/ATen/BatchingRegistrations.cpp

```diff
@@ -181,6 +181,11 @@ Tensor expand_batching_rule(const Tensor& self, IntArrayRef size, bool implicit)
   return self_physical.getPhysicalToLogicalMap().apply(result);
 }
 
+Tensor expand_batching_rule_symint(const Tensor& self, SymIntArrayRef psize, bool implicit) {
+  return expand_batching_rule(self, ExpectIntArrayRef(psize), implicit);
```

Comment:
it's a little weird for ExpectIntArrayRef to be capitalized cuz it's not actually a class right

Reply:
hehe that seemed to be a JIT convention. Most public TORCH_API is capitalized

Reply:
will fix